dispatcher/dispatcher.go

```diff
@@ -383,6 +396,15 @@ func (d *Dispatcher) processNextEvent() bool {
 				sleepTime = time.Millisecond * 10
 			}
 			time.Sleep(sleepTime)
+
+			if d.cfg.Real {
+				curTime := d.speedStartTime + uint64(float64(time.Since(d.speedStartRealTime)/time.Microsecond)*d.speed)
```

Comment:
Is having no space around mathematical operators the preferred style?

Reply:
It's formatted by `https://blog.golang.org/gofmt`. We don't have a choice here. 
One major benefit of Golang is that we don't argue about style preferences.

Reply:
I just read about it today! Still curious to see that it gives spaces to some operators (like `time.Millisecond * 10` and `d.speedStartTime + `) but no the others like `d.speedStartRealTime)/time.Microsecond)*d.speed`.

Reply:
This is intended. See also: https://github.com/golang/go/issues/1206#issuecomment-66052999